pvlib/solarposition.py

```diff
@@ -1445,3 +1453,80 @@ def sun_rise_set_transit_geometric(times, latitude, longitude, declination,
     sunset = _local_times_from_hours_since_midnight(times, sunset_hour)
     transit = _local_times_from_hours_since_midnight(times, transit_hour)
     return sunrise, sunset, transit
+
+
+def spencer(times, latitude, longitude):
+    """
+    Calculate the solar position using a formulation by Spencer 1971/1972.
+
+    Parameters
+    ----------
+    times : pandas.DatetimeIndex`
+        Corresponding timestamps, must be localized to the timezone for the
+        ``latitude`` and ``longitude``.
+    latitude : float
+        Latitude in degrees, positive north of equator, negative to south
+    longitude : float
+        Longitude in degrees, positive east of prime meridian, negative to west
+
+    Returns
+    -------
+    DataFrame
+        The DataFrame will have the following columns:
+        zenith (degrees),
+        elevation (degrees),
+        azimuth (degrees),
+        equation_of_time (seconds),
+        declination (degrees).
+
+    References
+    ----------
+    [1] Spencer (1972) can be found in
+    Sen, Zekai. Solar energy fundamentals and modeling techniques:
+     atmosphere, environment, climate change and renewable energy.
+     Springer Science & Business Media, 2008.
+    """
+
+    julians, julian_2000 = datetime_to_julian(times)
+    julians_2000 = np.asarray(julians, dtype=np.float) - julian_2000
+
+    latitude_radians = np.radians(latitude)
+    day_time = (julians_2000 % 1) * 24
+
+    declination = np.array(declination_spencer71(times.dayofyear))
+
+    # Equation of time (difference between standard time and solar time).
+    eot = np.array(equation_of_time_spencer71(times.dayofyear))
+
+    # True local time
+    tlt = (day_time + longitude / 15 + eot / 60) % 24 - 12
+
+    # Solar hour angle
+    ha = np.radians(tlt * 15)
+
+    # Calculate sun elevation.
+    sin_sun_elevation = (
+            np.sin(declination) * np.sin(latitude_radians) +
+            np.cos(declination) * np.cos(latitude_radians) * np.cos(ha)
+    )
+
+    # Compute the sun's elevation and zenith angle.
+    elevation = np.arcsin(sin_sun_elevation)
+    zenith = np.pi / 2 - elevation
+
+    # Compute the sun's azimuth angle.
+    y = -(np.sin(latitude_radians) * np.sin(elevation) - np.sin(declination)) \
```

Comment:
@meteoDaniel can you please replace these lines
https://github.com/pvlib/pvlib-python/blob/03bb14384d896b45b485ea88d70521f878d1e23a/pvlib/solarposition.py#L1517-L1524
with a call to `solar_azimuth_analytical(latitude, hourangle, declination, zenith)` ?